core/benches/banking_stage.rs

```diff
@@ -258,7 +259,13 @@ fn bench_banking(bencher: &mut Bencher, tx_type: TransactionType) {
                 }
                 verified_sender.send(v.to_vec()).unwrap();
             }
-            check_txs(&signal_receiver2, txes / CHUNKS);
+            // banking stage will drop duplicated transactions upon receiving, hence the test
+            // should not expect receiving anything if it starts to resend prepared transactions.
```

Comment:
I think you've changed what this bench is trying to do. We need to clear the LRU cache after each bench iteration

Reply:
I'll open a PR to address this since I already fixed it here: https://github.com/solana-labs/solana/pull/21749/files#diff-52a00f71d9630c25d4c08578554e73cd35fd4f51c332e623f7ce6522e384aebb

Reply:
> I'll open a PR to address this

awesome! so do i merge this PR first, or wait till your PR is ready?

Reply:
Let's wait if that's cool with you, the PR is ready here: https://github.com/solana-labs/solana/pull/22080

Reply:
I don't think you need these bench changes anymore